### PR TITLE
[DT-489][risk=no] Updating index to reflect PFHH as if a unified survey

### PR DESCRIPTION
**Pull request by freemabd**
Updating index to reflect PFHH as if a unified survey:

- Update matching question/answers across surveys
- remove old survey question/answers that are deprecated
- insert skip answers for new survey

---
**PR checklist**

- [ ] I have included an issue ID or "no ticket" in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [ ] I have included a risk tag of the form `[risk=no|low|moderate|severe]` in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [ ] I have manually tested this change and my testing process is described above.
- [ ] This change includes appropriate automated tests, and I have documented any behavior that cannot be tested with code.
- [ ] I have added explanatory comments where the logic is not obvious.
- One or more of the following is true:
  - [ ] This change is intended to complete a JIRA story, so I have checked that all AC are met for that story.
  - [ ] This change fixes a bug, so I have ensured the steps to reproduce are in the Jira ticket or provided above.
  - [ ] This change impacts deployment safety (e.g. removing/altering APIs which are in use), so I have documented the impacts in the description.
  - [ ] This change includes a new feature flag, so I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later.
  - [ ] This change modifies the UI, so I have taken screenshots or recordings of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P).
  - [ ] This change modifies API behavior, so I have run the relevant E2E tests locally because API changes are not covered by our PR checks.
  - [ ] None of the above apply to this change.

### CODE REVIEW

**Review by dolbeew — APPROVED** [review]

File: api/db-cdr/generate-cdr/test-data/pfhh_skip.sql [hunk 1]
```diff
@@ -0,0 +1,94 @@
+--this query helps build the prep_pfhh_non_answer_insert table
+--it contains all skip combos for each question
+INSERT INTO `all-of-us-ehr-dev.brian_test.prep_pfhh_non_answer_insert`
```
Comment (chenchals) [inline api/db-cdr/generate-cdr/test-data/pfhh_skip.sql:3]: Is this intentional?

File: api/db-cdr/generate-cdr/build-ds-tables.sh [hunk 2]
```diff
@@ -331,18 +331,17 @@ function do_PFHH(){
...
-                FROM \`$BQ_PROJECT.$BQ_DATASET.cb_criteria\`
-                WHERE concept_id IN (1740639)
-                AND domain_id = 'SURVEY'
-              ) a ON (c.path LIKE CONCAT('%', a.id, '.%'))
-          WHERE domain_id = 'SURVEY'
-          AND type = 'PPI'
-          AND subtype = 'ANSWER'
-      ) b on a.value_source_concept_id = b.answer_concept_id
+          SELECT *
+          FROM \`$BQ_PROJECT.$BQ_DATASET.R2022Q4R9.prep_concept_ancestor\`
```
Comment (chenchals) [inline api/db-cdr/generate-cdr/build-ds-tables.sh:335]: FROM \`$BQ_PROJECT.$BQ_DATASET.R2022Q4R9.prep_concept_ancestor\` seems incorrect dataset referred twice?

**Review by chenchals — CHANGES_REQUESTED** [review]

File: api/db-cdr/generate-cdr/test-data/pfhh_skip.sql (hunk 1 above)
Comment (freemabd) [reply api/db-cdr/generate-cdr/test-data/pfhh_skip.sql:3]: this is SQL used to create the insert table. It's manual SQL that I had to run per skip answer combos. This is located in the test scripts. I just added it incase we have to revisit this again.

File: api/db-cdr/generate-cdr/build-ds-tables.sh (hunk 2 above)
Comment (freemabd) [reply api/db-cdr/generate-cdr/build-ds-tables.sh:335]: fixed.

**Review by chenchals — APPROVED** [review]
Thanks

**Pull request merged**